ccp/src/main/java/com/togitech/ccp/component/TogiCountryCodePicker.kt

```diff
@@ -86,12 +95,18 @@ fun TogiCountryCodePicker(
     showPlaceholder: Boolean = true,
     includeOnly: ImmutableSet<String>? = null,
     clearIcon: ImageVector? = Icons.Filled.Clear,
+    keyboardOptions: KeyboardOptions = KeyboardOptions.Default.copy(
+        keyboardType = KeyboardType.Phone,
+        autoCorrect = true,
+        imeAction = ImeAction.Done,
+    ),
+    keyboardActions: KeyboardActions = KeyboardActions.Default,
```

Comment:
This is a behavior change since it's now missing the `onDone` block. Maybe this could default null and use the existing behavior in that case.

Reply:
>  Maybe this could default null and use the existing behavior in that case.

I will implement this and push changes